xml_analyzer/data_container.py

```diff
@@ -29,7 +39,7 @@ def set_constraint_specification(self, constraint_spec: str):
         self.constraint_specification = constraint_spec
 
     def get_variable_names(self) -> list:
```

Comment:
property as well - all setters and getters should be properties

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/6614ef2d31e5e3284ee434bbd113e446107a9606